notify/impl.go

```diff
@@ -934,6 +938,78 @@ func (n *Pushover) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	return false, nil
 }
 
+type Jira struct {
+	conf *config.JiraConfig
+	tmpl *template.Template
+}
+
+func NewJira(c *config.JiraConfig, t *template.Template) *Jira {
+	return &Jira{conf: c, tmpl: t}
+}
+
+type jiraCreateIssueRequest struct {
+	Fields map[string]interface{} `json:"fields"`
+}
+
+type jiraCreateIssueResponse struct {
+	ID   string `json:"id"`
+	Key  string `json:"key"`
+	Self string `json:"self"`
+}
+
+// Notify implements the Notifier interface.
+func (n *Jira) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
+	var err error
+	data := n.tmpl.Data(receiverName(ctx), groupLabels(ctx), as...)
+	tmpl := tmplText(n.tmpl, data, &err)
+
+	fields := map[string]interface{}{
+		"project":     map[string]string{"key": tmpl(n.conf.Project)},
+		"issuetype":   map[string]string{"name": tmpl(n.conf.IssueType)},
+		"summary":     tmpl(n.conf.Summary),
+		"description": tmpl(n.conf.Description),
+	}
+	// check errors from tmpl()
+	if err != nil {
+		return false, err
+	}
+	if n.conf.Priority != "" {
+		fields["priority"] = map[string]string{"name": n.conf.Priority}
+	}
+	for key, value := range n.conf.Fields {
+		fields[key] = value
+	}
+	body, err := json.Marshal(jiraCreateIssueRequest{Fields: fields})
+	if err != nil {
+		return false, err
+	}
+
+	req, err := http.NewRequest("POST", "https://jira.atlassian.com/rest/api/2/issue", bytes.NewBuffer(body))
```

Comment:
This will create a new JIRA every `group_interval`:
https://prometheus.io/docs/alerting/configuration/#route-route

I'm guessing it'll also create a new JIRA for every resolved alert notification.